packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -41,7 +49,7 @@ export const DefaultCell = (props: TableCellProps) => {
   }
 
   return (
-    <div {...cellProps} className={cellStyle}>
+    <div {...cellProps} onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave} className={cellStyle}>
```

Comment:
you should skip this if `showActions` is false

Reply:
Done